src/main/java/org/aksw/iguana/cc/storage/Storable.java

```diff
@@ -0,0 +1,36 @@
+package org.aksw.iguana.cc.storage;
+
+import org.apache.jena.rdf.model.Model;
+
+import java.util.List;
+
+public interface Storable {
+
+    record CSVData (
+            String folderName,
+            List<CSVFileData> files
+    ) {
+        public record CSVFileData(String filename, String[][] data) {}
+    }
+
+    interface AsCSV {
```

Comment:
toCSV is not used by CSVStorage. So, what is it good for?

Reply:
It is used [here](https://github.com/dice-group/IGUANA/blob/4f0a309a4b56e8059488ec16da2080fc42fbc21f/src/main/java/org/aksw/iguana/cc/storage/impl/CSVStorage.java#L192C19-L192C19)